### DISCUSSION TITLE
Population weighted residuals

### DISCUSSION THREAD

**Discussion by VishnuEriyat** [LAPKB/Pmetrics#206] [Q&A] [answered]
This I got while modelling data for rifampicin concentrations. The samples are collected serially at 0.5 hour intervals after dosing. The population weighted residuals in the right hand side is showing the residuals in a vertical column like fashion. I really could not understand why. Is there a brief explanation for this or any resource I can refer.
![Population predicted](https://github.com/LAPKB/Pmetrics/assets/153790648/77649ffd-3174-4f97-be53-ff24882a59ed)

**Comment by mnneely** [comment, marked as answer]
Looks like you have a lot of predictions of 0 when the observed values are not 0. Since observed is always ≥0, the residuals (pred - obs) are ≤0 when pred = 0. Perhaps try adding another compartment to your model?

**Reply by VishnuEriyat** [reply]
Thanks, will try that
